editor/src/uuiui/styles/utopi-color-helpers.ts

```diff
@@ -1,47 +1,34 @@
 import ChromaWrongTypes from 'chroma-js'
 const Chroma = ChromaWrongTypes as any
 
-export interface UtopiColor {
+export class UtopiColor {
   /**
-   * css-compatible rgba string: `rgba(255,0,255,1)` format
+   * css variable associated with this color: var(--variable-name) format
    */
-  value: string
-
+  value: string = 'var(--utopitheme-not-set)'
```

Comment:
We don't need to give these variables any values here since we're setting them in the constructor